dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -486,6 +488,26 @@ open class Jpql : JpqlDsl {
         return Expressions.times(this.toExpression(), value.toExpression())
     }
 
+    /**
+     * Creates an expression that represents  the current date.
+     *
+     * This is the same as ```CURRENT_DATE```.
+     */
+    @SinceJdsl("3.4.0")
+    fun currentDate(): Expression<LocalDate> {
+        return Expressions.currentDate()
+    }
```

Comment:
In the Criteria API, the return type of `currentDate()` is the `java.sql.Date`.

![image](https://github.com/line/kotlin-jdsl/assets/31586979/8e8b3a80-5d7f-4589-84c2-65a687cb9be1)

[The Jakarta specification](https://jakarta.ee/specifications/persistence/3.1/jakarta-persistence-spec-3.1.html#a5304) also specifies that the return type is the `java.sql.Date`. Therefore, please change the return type to the `java.sql.Date`.

![image](https://github.com/line/kotlin-jdsl/assets/31586979/5b213361-0027-47a3-8c33-449690ebc3e1)

Reply:
For the same reason as above, I would like to request that the return type of `currentTime()` be changed to the `java.sql.Time`.

Reply:
i got it, I'll change currentTime & currentDate